enterprise/app/BUILD.bazel

```diff
@@ -73,3 +73,10 @@ genrule(
         cat out > $@;
     """,
 )
+
+load("//rules/sha:index.bzl", "sha")
+
+sha(
+    name = "sha",
```

Comment:
uber nit: rename to `app_bundle_sha` since we probably also want `style_sha` in here as well at some point

Reply:
My plan is to ship all of the static files as a single bundle (when we add more than the js files).